README.md

```diff
@@ -27,3 +27,78 @@ You will need to configure the following environment variables and make them ava
 Also, for publishing to GitHub, you will need to provide a GitHub API key.
 
 For that, in travis, you can use the `env`. If your project is public, make sure to use [secrets](https://docs.travis-ci.com/user/encryption-keys/).
+
+## Tasks
+
+### Tests packages
+
+Three rules are available `test` plain execution of test, `test-race` execute
+the test with the `-race` flag and `test-coverage` it runs the tests with
+coverage support and generate the `coverage.txt` file. The coverage file can
+be uploaded to codecov using the rule `codecov`.
+
+### Building packages
+
+The rule `packages` creates the distribution packages, containing the command
+line utilities defined by the `COMMANDS` compiled for the architectures and
+OS, define by `PKG_OS` and `PKG_ARCH`.
+
+The package filename is based on the pattern: `<project>_<version>_<os>_<arch>.tar.gz`
+
+### Docker build and push
+
+The rule `docker-build` builds the defined dockerfiles by `DOCKERFILES`. Several
+dockerfile may be define per project eg.:
```

Comment:
s/dockerfile/dockerfiles/

s/may be define per project/may be defined in a project/